vuu/src/main/scala/org/finos/vuu/core/row/InMemMapRowBuilder.scala

```diff
@@ -0,0 +1,45 @@
+package org.finos.vuu.core.row
+import org.finos.vuu.core.table.{Column, RowData, RowWithData}
+
+import scala.collection.mutable
+
+class InMemMapRowBuilder extends RowBuilder {
+
+  private val mutableMap = new mutable.HashMap[String, Any]()
+  private var key: String = ""
+  override def setLong(column: Column, v: Long): RowBuilder = {
+    mutableMap.put(column.name, v)
```

Comment:
For the future: might be able to check for type consistency by matching the type i.e. `Long` with `column.dataType`.

Reply:
Yes, that was indeed my goal. So checking datatype of column matches call signature.
I will add that.